config.py

```diff
@@ -6,8 +6,10 @@
 
 THREADS_PER_PAGE = 8
 
-UPLOAD_PATH = '/mnt/adapt/.upload'
 STORE_PATH = '/mnt/adapt'
+UPLOAD_PATH = os.path.join(STORE_PATH, '.upload')
+
+MAX_CONTENT_LENGTH = (1 * 1024 * 1024) + (512 * 1024)
```

Comment:
ConteNt, not conteXt  (:  This is the maximum request size that Flask will accept.  Flask has to buffer the entire request and this short-circuits and rejects overly large requests, protecting the server from misbehaving clients.